### PR TITLE
Add CIRAL's mono regressions

### PR DESCRIPTION
**Pull request by Mofetoluwa**
Adding CIRAL's monolingual regressions for dev set (10 samples):
- Regressions for Hausa, Somali, Swahili and Yoruba
- Results successfully reproduced

### CODE REVIEW

**Comment by lintool** [comment]
hi @Mofetoluwa can you put the command-line here so I can try?

**Comment by Mofetoluwa** [comment]
Here they are:
```
nohup python src/main/python/run_regression.py --index --verify --search --regression ciral-hausa-dev >& logs/log.ciral-v1.0-mono-ha &
nohup python src/main/python/run_regression.py --index --verify --search --regression ciral-somali-dev >& logs/log.ciral-v1.0-mono-so &
nohup python src/main/python/run_regression.py --index --verify --search --regression ciral-swahili-dev >& logs/log.ciral-v1.0-mono-sw &
nohup python src/main/python/run_regression.py --index --verify --search --regression ciral-yoruba-dev >& logs/log.ciral-v1.0-mono-yo &
```

**Review by lintool — APPROVED** [review]
lgtm

please merge in `tools/` verify everything works, and then 🚢 it!

File: src/main/java/io/anserini/search/SearchCollection.java [hunk 1]
```diff
@@ -662,6 +662,8 @@ public Args searchTweets() {
 
   private Analyzer getAnalyzer() {
     try {
+      // Languages using Whitespace analyzer
```
Comment (lintool) [inline src/main/java/io/anserini/search/SearchCollection.java:665]: Remove these two commented out lines?

**Pull request merged**